ts/packages/cli/src/commands/export.ts

```diff
@@ -130,6 +146,38 @@ export default class Export extends BaseCommand<typeof Export> {
       description: `Transforms the casing of Config Keys in the export result to camelCase, PascalCase, Capital Case, snake_case, param-case, CONSTANT_CASE and others`,
       options: Object.keys(casingFormatters),
     }),
+    'pick-label': Flags.string({
+      description: `Pick a specific label from the previous eval command return to export`,
+      multiple: true,
+    }),
+    'omit-label': Flags.string({
+      description: `Omit a specific label from the previous eval command return to export`,
+      multiple: true,
+    }),
+    'pick-key': Flags.string({
+      description: `Pick a specific key from the previous eval command return to export`,
+      multiple: true,
+    }),
+    'omit-key': Flags.string({
+      description: `Omit a specific key from the previous eval command return to export`,
+      multiple: true,
+    }),
+    'pick-hidden': Flags.boolean({
+      description: `Explicitly include config keys marked as hidden. By default, hidden keys are omitted.`,
+      exclusive: ['omit-hidden'],
+    }),
+    'omit-hidden': Flags.boolean({
+      description: `Explicitly exclude config keys marked as hidden. By default, hidden keys are omitted.`,
+      exclusive: ['pick-hidden'],
+    }),
+    'pick-empty': Flags.boolean({
+      description: `Include config keys with empty values. By default, empty values are included.`,
+      exclusive: ['omit-empty'],
+    }),
+    'omit-empty': Flags.boolean({
+      description: `Exclude config keys with empty values. By default, empty values are included.`,
+      exclusive: ['pick-empty'],
+    }),
```

Comment:
Please remove the period at the end of the descriptions to align with the rest of the example descriptions. `format` for example has a period in the middle of the sentence but not at the end.